editor.planx.uk/src/pages/Preview/Node.tsx

```diff
@@ -239,8 +241,11 @@ const Node: React.FC<Props> = (props) => {
         />
       );
 
-    case TYPES.ExternalPortal:
     case TYPES.Filter:
+      return <FilterComponent {...getComponentProps<Filter>()} />;
```

Comment:
Minor styling bug happening here - the "(Flags Filter)" node isn't setting the `wasVisited` green border as it should - it's only getting set on the individual flag option. The Filter's node id is in the breadcrumbs as expected.

![Screenshot from 2024-10-06 20-09-36](https://github.com/user-attachments/assets/95f9e36a-4a0e-43a2-a046-a5ec5313f33e)

Reply:
I think I know what's causing this, happy to take a look here!

Reply:
Fixed in https://github.com/theopensystemslab/planx-new/pull/3869

Reply:
Approved & merged, thanks !